extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -675,24 +675,28 @@ public function getFormattedQuoteText() {
 		 * @var $mw WallMessage
 		 */
 		$mw = WallMessage::newFromId($messageId);
-		$mw->load();
+		if ( !empty( $mw ) ) {
+			$mw->load();
 
-		if(!empty($mw)) {
 			$username = $mw->getUser()->getName();
 
-			$convertToFormat = $this->request->getVal('convertToFormat', '');
+			$convertToFormat = $this->request->getVal( 'convertToFormat', '' );
 
-			if($convertToFormat == 'wikitext') {
-				$markup = '<div class="quote">' . "\n" . wfMsgForContent('wall-quote-author', $username) . "\n" . $mw->getRawText() . "\n</div>\n";
+			if ( $convertToFormat == 'wikitext' ) {
+				$markup = '<div class="quote">'
+					. "\n" . wfMsgForContent( 'wall-quote-author', $username )
```

Comment:
While your tidying up in here lets replace these deprecated wfMsg calls.  I think you could use:
```php
wfMessage( 'key', $args )->inContentLanguage()->escaped();
```